tools/mafft/mafft.xml

```diff
@@ -64,6 +68,10 @@
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
         </conditional>
+        <param name="adjustdirection" type="select" display="radio" label="Direction of nucleotide sequences" help="generate reverse complement sequences, as necessary, and align them together with the remaining sequences">
+            <option value="adjustdirection">adjust direction</option>
```

Comment:
Ok, nice!
Than my last concern. If you use `--adjustdirection` in the option tag inside of your select you can simply use `$adjustdirection` in the command section.